test/sys/test_pthread.rs

```diff
@@ -3,5 +3,5 @@ use nix::sys::pthread::*;
 #[test]
 fn test_pthread_self() {
     let tid = pthread_self();
-    assert!(tid > 0);
+    assert!(tid != 0);
```

Comment:
0 still isn't valid here, we should really fix this at some point, but this is fine for now.